koku/masu/database/trino_sql/reporting_awscostentrylineitem_daily_summary.sql

```diff
@@ -100,6 +101,7 @@ FROM (
         max(lineitem_blendedrate) as blended_rate,
         sum(lineitem_blendedcost) as blended_cost,
         sum(savingsplan_savingsplaneffectivecost) as savingsplan_effective_cost,
+        sum(calculated_amortized_cost) as calculated_amortized_cost,
```

Comment:
The next step we do here is to actually calculate `calculated_amortized_cost` as it isn't present on the `aws_line_items_daily` table, so this won't work as expected and I don't think we want to use the same name twice anyway since both lines are `as calculated_amortized_cost`.

Reply:
Hmm, I need to read this more carefully and understand what it's doing.

Reply:
Oh geez, I know what this is doing. This line is totally wrong. I'll remove it.